dist-test/run.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
-
```

Comment:
To handle failures in the worker and still be able to print logs we have to switch off `set -e`